axlearn/common/trainer.py

```diff
@@ -324,6 +325,7 @@ def __init__(
                     model=self.model,
                     model_param_partition_specs=model_param_partition_specs,
                 )
+        self._maybe_record_event(measurement.Event.END_ACCELERATOR_INIT)
```

Comment:
Can you clarify what accelerator init is supposed to capture? E.g., would `utils_spmd` where we call jax distributed init be more appropriate?

Reply:
This is supposed to capture device related initialization such as device scanning, mesh initialization, device reinit/reset, security setup, initialization of pre-mapped buffers etc. You are right, jax distributed init should be included here.

I would lean on your team to update/re-position the record calls to locations that seems best fit for this codebase.

Reply:
Thanks @dipannita08 ! WDYT about removing the `trainer.py` changes for now, so that we can add them in a follow-up PR? This PR can focus on adding the scaffolding in `measurement.py`.

Reply:
Sounds good, I'll circle back with you on those changes in a follow up PR.